src/store/ducks/generic/jurisdictions.ts

```diff
@@ -0,0 +1,290 @@
+import { get, keyBy, values } from 'lodash';
+import { AnyAction, Store } from 'redux';
+import SeamlessImmutable from 'seamless-immutable';
+
+/** the reducer name */
+export const reducerName = 'GenericJurisdictions';
+
+/** GenericJurisdiction interface */
+export interface GenericJurisdiction {
+  id: string;
+  jurisdiction_depth: number;
+  jurisdiction_id: string;
+  jurisdiction_name: string;
+  jurisdiction_name_path: string[];
+  jurisdiction_parent_id: string;
+  jurisdiction_path: string[];
+  plan_id: string;
+}
+
+export interface NamibiaJurisdiction extends GenericJurisdiction {
```

Comment:
Not sure if we should have this defined here.

Reply:
I think we either define them here as custom versions of the `GenericJurisdiction` interface, or import them from helper or typing files.